src/components/actions.ts

```diff
@@ -91,6 +94,22 @@ export const getAppointment = async (appointmentId: string | undefined) => {
   }
 };
 
+export const getPractice = async (practiceId: string | null | undefined) => {
```

Comment:
This function might be redundant since the the practice data is actually being retrieved in `[practiceCode]/book/page.tsx`

It should be possible to pass it down as a prop to the `<AppointmentForm/>` component

Reply:
Thanks for pointing that out @mentalcaries ! I've made adjustments for both emails so that there is no extra DB call to retrieve practice information.